File: 프로그래머스/0/120891. 369게임/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;
        
        String[] str = String.valueOf(order).split("");
        for(int i = 0; i < str.length; i++) {
            int now = Integer.parseInt(str[i]);
            if(now != 0 && now % 3 == 0) {
                answer++;
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120891. 369게임/369게임.kt
class Solution {
    fun solution(order: Int): Int {
        var answer: Int = 0
        var num = order
        
        while(num > 0) {
            val cur = num % 10;
            if(cur== 3 || cur == 6 || cur == 9) {
                answer++;
            }
            num /= 10
        }
        
        return answer
    }
}